tests/python/contrib/test_hexagon/test_software_pipeline_async.py

```diff
@@ -46,18 +47,33 @@ def plus_one_ref(a):
     return plus_one_primfunc, plus_one_ref
 
 
-@tvm.testing.requires_hexagon
-def test_software_pipeline_with_cache_read(hexagon_launcher, compute, outer, inner, dtype, scope):
+@tvm.testing.fixture
```

Comment:
is `requires_hexagon` now implicit?

Reply:
This is an artifact of the GitHub diff.  I split out the schedule as a parameter here.  The test now starts on line 74.